### DISCUSSION TITLE
Do I need Coral with a Beelink EQ13

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#13796] [Ask A Question]
I'm looking to get frigate set up with HA. I have 10 IP cameras and I ordered a Beelink EQ13. 

In the meantime, I was curious if you all felt I definitely will need a Coral as well. I don't have a need for detection ATM. I'm mainly just interested in 24/7 recording, as I already have a Ring camera for the one area where I need detection.

So I'm not sure if a Coral will add much value. 

Thoughts?

**Comment by NickM-27** [comment]
You can just use openvino for detection
